tests/conftest.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 
 from horse import build_app
+from horse.models.movie import Movie
+from horse.models.user import User
```

Comment:
we can hide the `movie` and `user` modules by importing all the models iin `horse.models.__init__`